entity_plus.module

```diff
@@ -1094,3 +1094,47 @@ function _entity_plus_modules_get_default_types($modules) {
   }
   return $types;
 }
+
+/**
+ * Implements hook_modules_disabled().
+ */
+function entity_plus_modules_disabled($modules) {
+  foreach (_entity_plus_modules_get_default_types($modules) as $entity_type) {
+    $info = entity_get_info($entity_type);
+
+    // Do nothing if the module providing the entity type has been disabled too.
+    if (isset($info['module']) && in_array($info['module'], $modules)) {
+      return;
+    }
+
+    $keys = $info['entity keys'] + array('module' => 'module', 'status' => 'status', 'name' => $info['entity keys']['id']);
```

Comment:
Small "coding standard" detail: please break each element in this array into its own line, as in: 

```php
array(
  'module' => 'module',
  'status' => 'status',
  'name' => $info['entity keys']['id']
);
```

This line is making the automatic test fail.